src/components/clusterWizard/ClusterWizardForm.tsx

```diff
@@ -89,63 +103,131 @@ const ClusterWizardForm: React.FC<ClusterWizardFormProps> = ({ cluster, setStep
                     <TextContent>
                       <Text component="h1">Configure a bare metal OpenShift cluster</Text>
                     </TextContent>
+                    <InputField label="Cluster name" name="name" isRequired />
+                    <SelectField
+                      label="OpenShift Version"
+                      name="openshiftVersion"
+                      options={[
+                        { label: 'OpenShift 4.4', value: '4.4' },
```

Comment:
Can we externalize these constants? And use the first of them to feed the default of initalValue in `ClusterWizardForm.tsx`?